integration_tests/poetry.lock

```diff
@@ -770,7 +770,7 @@ tomlkit = "^0.7.0"
 type = "git"
 url = "https://github.com/crypto-com/pystarport.git"
 reference = "main"
-resolved_reference = "286a446473e1f3e1e1a5e075ba1041d3b52e5fc9"
+resolved_reference = "eab5b1ad06a53fae6586bea3dcc0f0cd2a1c4682"
```

Comment:
Update pystarport is for this change?
https://github.com/crypto-com/pystarport/pull/66/files

Reply:
yup but need fix ica test due to the strategy change